libraries/botbuilder-core/src/activityHandler.ts

```diff
@@ -694,7 +694,7 @@ export class ActivityHandler extends ActivityHandlerBase {
     }
 
     private getAdaptiveCardInvokeValue(activity: Activity): AdaptiveCardInvokeValue {
-        const { value } = activity;
+        const { value }: { value?: AdaptiveCardInvokeValue } = activity;
```

Comment:
Is this explicit type necessary? I would have thought it was inferred?

Reply:
Necessary I think because `activity.value` can be anything.

Note: Had to use optional `value?` because otherwise eslint complains that it can't cast because `activity.value` is optional.

![image](https://user-images.githubusercontent.com/40401643/122620657-607d9e00-d048-11eb-80e1-dc413b496162.png)
![image](https://user-images.githubusercontent.com/40401643/122620665-65dae880-d048-11eb-87b5-1792f6a33405.png)